### ISSUE TITLE
Add placeholder icon for .docxf files

### ISSUE THREAD

**Issue by jesmrec** [owncloud/android#4267]
.docxf format will be supported by oCIS (open-with-web), so it will be usual to find such kind of files in the lists.

We should assign it a thumbnail-placeholder that is not the default one.

![Screenshot 2023-12-18 at 12 59 12](https://github.com/owncloud/android/assets/14894746/7ee11506-6196-4ec4-8a52-4ed44d254bf6)

### TASKS

- [ ] Research (if needed)
- [ ] Create branch feature/feature_name
- [ ] Development tasks
  - [ ] Implement whatever
  - [ ] ...
- [ ] Code review and apply changes requested
- [ ] Design test plan
- [ ] QA
- [ ] Merge branch feature/feature_name into master

**Comment by kulmann** [comment]
oC Web introduced an icon mapping in https://github.com/owncloud/web/pull/10037 and the icon lives here https://github.com/owncloud/web/blob/master/packages/design-system/src/assets/icons/resource-type-form-fill.svg

**Comment by Aitorbp** [comment]
I send you a screenshot of the `docxf` extension icon taken from material. It's right? @michaelstingl @kulmann @jesmrec @JuancaG05 

![Image](https://github.com/owncloud/android/assets/34248951/da39c6e4-1e7f-4018-9801-2dcf93dac7dc)

**Comment by michaelstingl** [comment]
> I send you a screenshot of the `docxf` extension icon taken from material. It's right? @michaelstingl @kulmann @jesmrec @JuancaG05

Could you post a link to the Google dev docs with the overview page of all available file type icons?

**Comment by kulmann** [comment]
> Icons from [remixicon.com](https://remixicon.com/)?

No. Usually our file type icons are taken from font awesome v5. But in this case I think @tbsbdr made custom ones.

**Comment by Aitorbp** [comment]
> > I send you a screenshot of the `docxf` extension icon taken from material. It's right? @michaelstingl @kulmann @jesmrec @JuancaG05
> 
> Could you post a link to the Google dev docs with the overview page of all available file type icons?

Hete you have: https://fonts.google.com/icons
<img width="823" alt="Captura de pantalla 2024-01-24 a las 8 37 53" src="https://github.com/owncloud/android/assets/34248951/1fe8bf8d-baaa-4a2d-9cb1-a04484e5d13a">

**Comment by tbsbdr** [comment]
@michaelstingl I can fiddle together this file (https://github.com/owncloud/web/blob/master/packages/web-pkg/src/helpers/resource/icon.ts) for you so that you get:
fileextensions, color and icon.
would that be helpful for you?

**Comment by Aitorbp** [comment]
I have changed the color of the icon to the same one that is on the website.
![new_image_docxf](https://github.com/owncloud/android/assets/34248951/5ed17cef-591d-491a-8121-b7edf62d6aa2)
